base_access_wallet.sol

```diff
@@ -178,8 +179,6 @@ contract BaseAccessWalletFactory is Ownable {
     bytes32 public version ="BsAccWltFactory20190506154200ML"; //class name (max 16), date YYYYMMDD, time HHMMSS and Developer initials XX
```

Comment:
Is it necessary to update version of baseAccessWalletFactory?

Reply:
Actually both version strings should be updated. Thanks.

Reply:
Wait, no. The version string for the `BaseAccessControlGroup` should be updated. Just did that.
This PR didn't really end up changing `BaseAccessWallet` or its factory.

Reply:
got it